beta-src/src/data/map/land/LandTerritoriesMapData.ts

```diff
@@ -35,6 +35,7 @@ export const ANKARA: TerritoryMapData = new TerritoryMapDataGenerator({
     x: 60,
     y: 100,
   },
+  fill: Texture.TURKEY,
```

Comment:
We don't want to set the initial fill to a map highlight, you can use the `texture` property to specific initial texture. Take a look at the interface.

Reply:
Would the following be okay or should areas that be be captured not have a fill defined? 

```
fill: Texture.LAND,
...
texture: {
 texture: Texture.TURKEY
}
```

Reply:
Anything defined in this file is for the initial map load. No territory is captured initially until we get the API which shows which countries own what territories.